package/upgrade/migrations/managed_charts/v1.1.0.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+
+CHART_NAME=$1
+CHART_MANIFEST=$2
+
+patch_harvester_ignore_default_sc()
+{
+	# add ignoring resources when upgrading to match the pr (https://github.com/harvester/harvester-installer/pull/481)
+	yq e '.spec.diff.comparePatches = [{"apiVersion": "storage.k8s.io/v1", "kind": "StorageClass", "name": "harvester-longhorn", "jsonPointers":["/metadata/annotations"]}]' $CHART_MANIFEST -i
```

Comment:
We need to append the new one to the list. This removes existing comparePatches and shows the previous modified state again:

```
    nonReadyResources:
    - bundleState: Modified
      modifiedStatus:
      - apiVersion: apiextensions.k8s.io/v1
        kind: CustomResourceDefinition
        name: engineimages.longhorn.io
        patch: '{"status":{"acceptedNames":{"kind":"","plural":""},"conditions":[],"storedVersions":[]}}'
      - apiVersion: apiextensions.k8s.io/v1
        kind: CustomResourceDefinition
        name: nodes.longhorn.io
        patch: '{"status":{"acceptedNames":{"kind":"","plural":""},"conditions":[],"storedVersions":[]}}'
      - apiVersion: apiextensions.k8s.io/v1
        kind: CustomResourceDefinition
        name: volumes.longhorn.io
        patch: '{"status":{"acceptedNames":{"kind":"","plural":""},"conditions":[],"storedVersions":[]}}'
  
```